### PR TITLE
changed "restart now" to "restart Github Desktop now"

### LINKED ISSUE #4891: Offering to 'restart GitHub now' instead of offering to 'restart now' when updates are available, can be confusing.
Offering to restart now discourages people who misinterpret restarting as restarting their computer to update GitHub (the desktop app) as they are in the middle of doing something. But offering instead to 'restart GitHub now', encourages those (most people) who are not using GiHub but still using their computer to restart GitHub to receive the new update.

### PR DESCRIPTION
**Pull request by SensehacK**
fixed #4891 with different line for text.

### CODE REVIEW

**Comment by iAmWillShepherd** [comment]
@SensehacK looks like you need to run the linter. Executing `yarn lint:fix` should do the trick 👍

**Comment by iAmWillShepherd** [comment]
I'm 👍  on the text change, but I'd like to keep this open for thoughts from @desktop/core.

File: app/src/ui/updates/update-available.tsx [hunk 1]
```diff
@@ -25,7 +25,8 @@ export class UpdateAvailable extends React.Component<
           An updated version of GitHub Desktop is available and will be
           installed at the next launch. See{' '}
           <LinkButton uri={this.props.releaseNotesLink}>what's new</LinkButton>{' '}
-          or <LinkButton onClick={this.updateNow}>restart now</LinkButton>.
+          or <LinkButton onClick={this.updateNow}>restart Github
```
Comment (donokuda) [inline app/src/ui/updates/update-available.tsx]: Thoughts on changing this to "restart GitHub Desktop" instead?

The two minor changes are capitalizing the "H" in GitHub and I don't think the "now" is necessary (although I don't strongly feel either way.)

**Comment by say25** [comment]
This is a duplicate PR of https://github.com/desktop/desktop/pull/4895

**Comment by shiftkey** [comment]
@SensehacK thanks for the contribution, but we're already discussing this similar change in #4895

**Comment by SensehacK** [comment]
@shiftkey  Sure I'm still in learning phase for Typescript & electron framework.

Will try to contribute in future :)

**Pull request closed without merging**